.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+# Description
+
+<!--Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.-->
+
+Fixes # (issue)<!--Add issue no. which you solved at place of issue-->
+
+## Type of change
+
+<!---Please delete options that are not relevant.-->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+# How Has This Been Tested?
+
+<!--Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration-->
```

Comment:
add a checklist for attaching screenshot here.

Reply:
> add a checklist for attaching screenshot here.


 - [ ] added screenshot of changes it makes .

like this? or should i write in comment that add screenshots of the changes you made

Reply:
Yes, exactly.